platform/view/sdk/sdk.go

```diff
@@ -238,18 +237,18 @@ func (p *p) initGRPCServer() error {
 		grpclogging.StreamServerInterceptor(flogging.MustGetLogger("comm.grpc.server").Zap()),
 	)
 
-	cs := grpc2.NewCredentialSupport()
-	if serverConfig.SecOpts.UseTLS {
-		logger.Info("Starting peer with TLS enabled")
-		cs = grpc2.NewCredentialSupport(serverConfig.SecOpts.ServerRootCAs...)
-
-		// set the cert to use if client auth is requested by remote endpoints
-		clientCert, err := p.getClientCertificate()
-		if err != nil {
-			logger.Fatalf("Failed to set TLS client certificate (%s)", err)
-		}
-		cs.SetClientCertificate(clientCert)
-	}
+	//cs := grpc2.NewCredentialSupport()
```

Comment:
This doesn't seem to be needed anywhere

Reply:
wired - we are not using `cs` anywhere. @angelo are we missing here something or is just a left over from the past?

Reply:
as discussed offline, let's remove